app/kubernetes-ingress-controller/2.0.x/guides/using-udpingress.md

```diff
@@ -0,0 +1,351 @@
+---
+title: UDPIngress with Kong
+---
+
+This guide walks you through deploying a simple [Service][svc] that
+listens for [UDP datagrams][udp], and exposes this service outside
+of the cluster using Kong Gateway.
+
+[svc]:https://kubernetes.io/docs/concepts/services-networking/service/
+[udp]:https://datatracker.ietf.org/doc/html/rfc768
+
+## Overview
+
+Some of the most common UDP-based services available on the internet are
+[DNS servers][dns]. These servers are an important part of the infrastructure
+of the internet, and are also [present by default][kubedns] in Kubernetes
+clusters to allow [Pods][pods] within the cluster to look up other pods' IP
+addresses by name.
+
+For this example, you will:
+* Deploy your own [CoreDNS][coredns] server, which is the default
+DNS server Kubernetes uses for internal DNS
+* Deploy a CoreDNS `Pod` and `Service`
+* Route UDP traffic to it using `UDPIngress`
+
+This guide assumes that you've deployed the Kong Kubernetes Ingress Controller (KIC)
+using the [Helm Chart][chart]. If you have deployed the KIC in a different way, you
+may need to make some manual adjustments to some of the provided instructions.
+
+[dns]:https://datatracker.ietf.org/doc/html/rfc1035
+[kubedns]:https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
+[pods]:https://kubernetes.io/docs/concepts/workloads/pods/
+[chart]:https://github.com/kong/charts
+
+## Installation
+
+Follow the [deployment](/kubernetes-ingress-controller/{{page.kong_version}}/deployment/overview) documentation to install
+the {{site.kic_product_name}} on your Kubernetes cluster.
+
+> **Note**: This feature is compatible with:
+> * Kong Gateway versions FIXME and above.
```

Comment:
Do we know which versions of the gateway this is compatible with yet, or does that still need to be tested?

Reply:
requires 2.0.0+, fix in 1513aa547490817ee8ea05712cda81025c216b91 :+1: